.github/workflows/publish-extension-nightly.yml

```diff
@@ -79,7 +79,16 @@ jobs:
         env:
           VSCE_PAT: ${{ secrets.VSCODE_PUBLISH_PWD }}
         working-directory: ./packages/plugin-core
-        run: yarn deploy:vscode
+        run: |
+          npx verdaccio > verdaccio.log  2>&1 &
+          sleep 30
+          echo "checking verdaccio log"
+          cat verdaccio.log
+          echo "Setting registry to local"
+          yarn config set registry http://localhost:4873
+          npm set registry http://localhost:4873/
+          echo "running deploy:vscode"
+          yarn deploy:vscode
 
       - name: Publish to Open VSX Marketplace
```

Comment:
does open vsx marketplace need same fix?

Reply:
This is a good question... in the pipeline that passed, it didn't seem to be a problem - but I think I'll at least set the reg keys again to be safe, as it seems like it _should_ require the vars to be set again.